modules/programs/bat.nix

```diff
@@ -60,6 +60,24 @@ in {
       '';
     };
 
+    syntaxes = mkOption {
+      type = types.attrsOf types.lines;
+      default = { };
+      example = literalExpression ''
+        {
+          syntaxes.gleam = builtins.readFile (pkgs.fetchFromGitHub {
+            owner = "molnarmark";
+            repo = "sublime-gleam";
+            rev = "2e761cdb1a87539d827987f997a20a35efd68aa9";
+            sha256 = "sha256-Zj2DKTcO1t9g18qsNKtpHKElbRSc9nBRE2QBzRn9+qs=";
```

Comment:
```nix
            hash = "sha256-Zj2DKTcO1t9g18qsNKtpHKElbRSc9nBRE2QBzRn9+qs=";
```
this is the recommended option in nixpkgs (but just a small nitpick)

Reply:
have changed this